app/templates/reward-page.html

```diff
@@ -9,7 +9,7 @@
     </div>
 
     <div ng-hide="reward.error">
-        <div class="content" ng-class="{active: reward.status == 'incomplete' || reward.status == 'pending' || reward.status == 'unverified' || reward.status == 'ineligible' || reward.status == 'updated_too_recently'}">
+        <div class="content" ng-class="{active: reward.status == 'incomplete' || reward.status == 'pending' || reward.status == 'unverified' || reward.status == 'ineligible' || reward.status == 'updated_too_recently' || reward.status == 'fb_email_missing'}">
```

Comment:
With the length of this, it seems like using `{active: reward.error}` would be preferable. Is there some reason not to?